bin/lima-and-qemu.pl

```diff
@@ -219,12 +189,237 @@ sub record {
             $filename = $name;
         }
     }
-    # Use human readable size of the file as the comment:
-    # $ ls -lh /usr/local/Cellar/libssh/0.9.5_1/lib/libssh.4.8.6.dylib
-    # -rw-r--r--  1 jan  staff   394K  5 Jan 11:04 /usr/local/Cellar/libssh/0.9.5_1/lib/libssh.4.8.6.dylib
     $deps{$filename} = sprintf "[%s]", (split / +/, qx(ls -lh $filename))[4];
 }
 
 sub dirname {
     shift =~ s|/[^/]+$||r;
-}
\ No newline at end of file
+}
+
+sub verify_dependencies {
```

Comment:
What's the purpose of verification? We know these file's version's change frequently. This seems like something that will constantly break and need updating